validator/testdata/feature_tests/amp-alaantv-breakingnews.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
```

Comment:
Could you move these into the extension directory instead of feature_tests? The format would be
extensions/<extension_name>/0.1/test/validator-<extension_name>.(html|out)

Here's more info on validator contributions: https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.md